hail/python/test/hailtop/batch/test_batch.py

```diff
@@ -521,3 +542,24 @@ def test_envvar(self):
         j.env('SOME_VARIABLE', '123abcdef')
         j.command('[ $SOME_VARIABLE = "123abcdef" ]')
         assert b.run().status()['state'] == 'success'
+
+    def test_single_job_with_shell(self):
+        with tempfile.NamedTemporaryFile('w') as output_file:
+            msg = 'hello world'
+
+            b = self.batch()
+            j = b.new_job(shell='bin/sh')
+            j.command(f'echo "{msg}" > {j.ofile}')
+
+            b.write_output(j.ofile, output_file.name)
```

Comment:
`output_file.name` is a local file, you can't do that.  Much easier to just echo and check the output.

Reply:
What's the easiest way to read stdout?